core/iwasm/aot/arch/aot_reloc_riscv.c

```diff
@@ -101,6 +106,8 @@ void __umoddi3(void);
 void __umodsi3(void);
 void __unorddf2(void);
 void __unordsf2(void);
+void __atomic_compare_exchange_4(void);
+void __atomic_store_4(void);
```

Comment:
The build_iwasm_on_nuttx CI reported errors:
![image](https://github.com/user-attachments/assets/c3122a40-641d-473e-8daa-40aea12ea60b)
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/11052682397/job/30739828374

How about changing to below lines to have a try:
```C
bool __atomic_compare_exchange_4(volatile void *, void *, unsigned int,
                                 bool, int, int);
void __atomic_store_4(volatile void *, unsigned int, int);
```

Reply:
Sure, I'll try it.